TreeTracker/UI/Navigation/Home/HomeCoordinator.swift

```diff
@@ -130,6 +159,58 @@ private extension HomeCoordinator {
         return viewcontroller
     }
 
+    func chatListViewController(planter: Planter) -> UIViewController {
+        let viewcontroller = StoryboardScene.ChatList.initialScene.instantiate()
+        viewcontroller.viewModel = {
+            let viewModel = ChatListViewModel(
+            planter: planter,
+            selfieService: self.treetrackerSDK.selfieService,
+            messagingService: self.treetrackerSDK.messagingService
+            )
+            viewModel.coordinatorDelegate = self
+            return viewModel
+        }()
+        return viewcontroller
+    }
+
+    func messagesViewController(planter: Planter) -> UIViewController {
+        let viewcontroller = StoryboardScene.Messages.initialScene.instantiate()
+        viewcontroller.viewModel = {
+            let viewModel = MessagesViewModel(
+            planter: planter,
+            messagingService: treetrackerSDK.messagingService
+            )
+            return viewModel
+        }()
+        return viewcontroller
+    }
+
+    func announceViewController(chat: ChatListViewModel.Chat) -> UIViewController {
+        let viewController = StoryboardScene.Announce.initialScene.instantiate()
+        viewController.viewModel = {
+            let viewModel = AnnounceViewModel(
+                chat: chat
+            )
+            return viewModel
+        }()
+        return viewController
+    }
+
+    func surveyViewController(planter: Planter, survey: SurveyViewModel.Survey, index: Int) -> UIViewController {
+        let viewController = StoryboardScene.Survey.initialScene.instantiate()
+        viewController.viewModel = {
+            let viewModel = SurveyViewModel(
+                planter: planter,
+                survey: survey,
+                messagingService: self.treetrackerSDK.messagingService,
+                index: index
```

Comment:
Looking at this method signature im not sure what "index" refers to, i think we should rename it to something more descriptive

Reply:
You're right.  We can change it to "questionNumber", it will be better even for the methods and logic that use this variable

Reply:
What do you think @AlexCornforth?